autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -242,26 +243,48 @@ def handle(self, *args, **options): # pylint: disable=too-many-locals, too-many-
                 meetings.extend(meetings_list)
 
         start = time.time()
-        for slc in slice_every(instructors, 50_000):
-            Instructor.objects.bulk_create(slc, ignore_conflicts=True)
+        Instructor.objects.bulk_create(instructors,
+                                       ignore_conflicts=True, batch_size=50_000)
```

Comment:
Instructors only store email addresses now (which we don't actually use anywhere) so it doesn't matter right now, but it's worth pointing out that this doesn't actually update existing instructors, do you think it's worth updating them

Reply:
Yeah I intentionally didn't update them as I didn't think it really mattered since we don't use the emails. I can if you'd like but I don't think it's necessary really

Reply:
You can leave it as is, we can change it if we actually start using emails for something but for now this works